lib/services/googleV2/index.ts

```diff
@@ -5,8 +5,14 @@ import Handler from './request';
 
 @injectServices({ handler: Handler })
 class GoogleManager extends AbstractManager<{ handler: Handler }> {
+  /** @deprecated */
   static SLOT_FILLING_PREFIX = 'slot_filling_';
 
+  static SLOT_FILLING_PREFIX_V2 = 'SLOT_FILLING_';
```

Comment:
would it be smart to instead make this
```suggestion
  static SLOT_FILLING_PREFIX_V2 = 'slot_filling_v2_';
```
in the event of a v3

Reply:
it just looks jarring on people's projects if they ever decide to go look at it
![Screen Shot 2021-12-17 at 9 53 00 AM](https://user-images.githubusercontent.com/5643574/146562866-f0b64435-81ba-4661-9561-86e72c5657c2.png)